frontend/src/js/concept-trees/globalTreeStoreHelper.ts

```diff
@@ -49,9 +49,9 @@ export function setTree(
 export function getConceptById(
   conceptId: ConceptIdT,
   rootId?: ConceptIdT,
-): ConceptT | null {
+): ConceptT | undefined {
```

Comment:
Refactored the global helper to return undefined, makes for easier handling of the return value in other places

Reply:
Is `ConceptT?` not supported by TS or is it something different in this context?

Reply:
`ConceptT?` is not valid syntax as far as I know